documentation/assemblies/overview/assembly-kafka-connect-components.adoc

```diff
@@ -0,0 +1,31 @@
+// This assembly is included in:
+//
+// overview/overview.adoc
+
+[id="kafka-connect-components_{context}"]
+= About Kafka Connect
+
+[role="_abstract"]
+Kafka Connect is an integration toolkit for streaming data between Kafka brokers and other systems.
+The other system is typically an external data source or target, such as a database.
+
+Kafka Connect uses a plugin architecture.
+Plugins allow connections to other systems and provide additional configuration to manipulate data.
+Plugins include _connectors_ and other components, such as data converters and transforms.
+A connector operates with a specific type of external system.
+Each connector defines a schema for its configuration.
+You supply the configuration to Kafka Connect to create a _connector instance_ within Kafka Connect.
+Connector instances define a set of tasks to move data between systems.
+
+Strimzi operates Kafka Connect in _distributed mode_, distributing data streaming tasks across one or more worker pods.
+A Kafka Connect cluster comprises a group of worker pods.
+A single worker instantiates a connector.
+Each connector comprises one or more tasks that are distributed across the group of workers.
+Distribution across workers creates highly scalable pipelines.
+
+Workers convert data into the correct format for the target system.
```

Comment:
A bit comma-laden, perhaps...

```suggestion
Workers convert data from, or to, the correct format for the source, or target, system.
```

Reply:
Not sure about _from the correct format_ . How about:
_Workers convert data from one format into another format that's suitable for the source or target system._